ods_ci/tests/Resources/CLI/MustGather/get-must-gather-logs.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 # Redirecting stdout/stderr of must-gather to a file, as it fills up the
 # process buffer and prevents the script from running further.
-oc adm must-gather --image=quay.io/modh/must-gather@sha256:1bd8735d715b624c1eaf484454b0d6d400a334d8cbba47f99883626f36e96657 &> must-gather-results.txt
+oc adm must-gather --image=quay.io/modh/must-gather@sha256:a57ac193de3a7e258d22eda528c598ef6ffe52725b09f226e0cc948b3b75bfb3 &> must-gather-results.txt
```

Comment:
This is wrong image - it's for RHOAI 2.11, please use this one:
```suggestion
oc adm must-gather --image=quay.io/modh/must-gather@sha256:9d5988f45c3b00ec7fbbe7a8a86cc149a2768c9c47e207694fdb6e87ef44adf3 &> must-gather-results.txt
```

You can check via https://quay.io/repository/modh/must-gather?tab=tags&tag=latest or https://github.com/red-hat-data-services/rhoai-disconnected-install-helper/blob/b72fd3c7fa50a67789698ae0e573d6438a4c6c7d/rhoai-2.13.md#L55